src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Testing/XUnit/XUnitIntegration.cs

```diff
@@ -17,7 +23,7 @@ internal static Scope CreateScope(ref TestRunnerStruct runnerInstance, Type targ
 
             string testFramework = "xUnit " + testInvokerAssemblyName.Version.ToString();
 
-            Scope scope = Common.TestTracer.StartActive("xunit.test", serviceName: Common.ServiceName);
+            Scope scope = Common.TestTracer.StartActive("xunit.test", serviceName: Common.TestTracer.DefaultServiceName);
```

Comment:
Are there any remaining uses of `Common.ServiceName`? Maybe we can remove that? Or redefine the property to always return `TestTracer.DefaultServiceName`?

Reply:
My apologies, the GitHub UI didn't refresh so I didn't see the latest code. Looks like you already did what I was asking! Resolving.